### ISSUE TITLE
RestrictImports doesn't handle multiple whitespaces well

### ISSUE THREAD

**Issue by dbwiddis** [skuzzle/restrict-imports-enforcer-rule#52]
This works:
```xml
<allowedImport>static org.junit.jupiter.api.**</allowedImport>
```
This (thanks to formatter wrapping long lines) fails:
```xml
<allowedImport>static 
    org.junit.jupiter.api.**</allowedImport>
```
> [INFO] --- maven-enforcer-plugin:3.0.0:enforce (ban-transitive-imports) @ oshi-parent ---
[WARNING] Rule 0: org.apache.maven.plugins.enforcer.RestrictImports failed with message:
Encountered unexpected exception: class java.lang.String cannot be cast to class de.skuzzle.enforcer.restrictimports.analyze.PackagePattern (java.lang.String is in module java.base of loader 'bootstrap'; de.skuzzle.enforcer.restrictimports.analyze.PackagePattern is in unnamed module of loader org.codehaus.plexus.classworlds.realm.ClassRealm @328d044f)

This also fails (two spaces):
```xml
<allowedImport>static  org.junit.jupiter.api.**</allowedImport>
```
Since these rules are all package-based and there aren't supposed to be spaces in package names, this seems to be entirely a side effect of having to include "static " with a single space.

**Comment by skuzzle** [comment]
Nice finding. I have some further changes coming up for version 2.0.0 and I will include a bugfix for this for sure

**Comment by skuzzle** [comment]
Will be fixed in 2.0.0 release
